x/accounts/accountstd/exports.go

```diff
@@ -13,7 +14,10 @@ import (
 	"github.com/cosmos/cosmos-sdk/types/address"
 )
 
-var accountsModuleAddress = address.Module("accounts")
+var (
+	accountsModuleAddress = address.Module("accounts")
```

Comment:
nit: how do we keep this in sync with x/accounts/ModuleName?

Reply:
eh good Q, it would be circular dep :\